usaspending_api/common/helpers/sql_helpers.py

```diff
@@ -19,11 +19,11 @@ def build_dsn_string(db_settings):
     return "postgres://{USER}:{PASSWORD}@{HOST}:{PORT}/{NAME}".format(**db_settings)
 
 
-def get_database_dsn_string():
-    if DEFAULT_DB_ALIAS in settings.DATABASES:
-        return build_dsn_string(settings.DATABASES[DEFAULT_DB_ALIAS])
+def get_database_dsn_string(db_alias=DEFAULT_DB_ALIAS):
+    if db_alias in settings.DATABASES:
+        return build_dsn_string(settings.DATABASES[db_alias])
     else:
-        raise Exception("No valid database connection is configured")
+        raise Exception(f'No valid database connection is configured with alias "{db_alias}"')
```

Comment:
What if db alias is not provided. Do we really want to output a message saying "No database connection configured with alias None".

Perhaps you should output a message saying "db alias must be provided and cannot be None"

Reply:
Done